.github/workflows/deploy.yml

```diff
@@ -41,4 +81,27 @@ jobs:
       - name: Install SAM CLI
         run: script/github-action-awscli-installation
       - name: Deploy the Serverless Demo App
-        run: script/deploy-canary-demo
+        run: script/deploy-canary-demo-beta
+  deploy_gamma_canary:
+    name: Deploy Demo App on Gamma Canary
+    runs-on: ubuntu-latest
+    env:
+        GAMMA_AWS_ACCESS_KEY_ID: ${{secrets.GAMMA_AWS_ACCESS_KEY}}
+        GAMMA_AWS_SECRET_ACCESS_KEY: ${{secrets.GAMMA_AWS_SECRET_ACCESS_KEY}}
+        AWS_DEFAULT_REGION: us-east-1
+        AWS_DEFAULT_OUTPUT: text
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
```

Comment:
Should these use `${{secrets.GAMMA_AWS_ACCESS_KEY}}` and `${{secrets.GAMMA_AWS_SECRET_ACCESS_KEY}}`?

Reply:
Similarly for the beta case as well.

Reply:
So the previous script was also not using these but still present i was not sure what they were for so kept it will remove it now.

Reply:
I am not sure which credentials we should used for running the script.

Reply:
```
echo "Deploying to beta stage for canary that talks to gamma Chime endpoint"
export AWS_ACCESS_KEY_ID=$BETA_AWS_ACCESS_KEY_ID
export AWS_SECRET_ACCESS_KEY=$BETA_AWS_SECRET_ACCESS_KEY
```

I think the deploy-canary-demo script exports these env variables before running the deploy script. I wonder if we need this step at all.